src/reducer/admin/notification.spec.js

```diff
@@ -7,22 +7,29 @@ import reducer from './notification';
 
 describe('notification reducer', () => {
     it('should return empty notification by default', () => {
```

Comment:
you should add one test showing that a custom `autoHideDuration` in the payload updates the state accordingly

Reply:
ok, added